ca.potatoe.cmdjewel.yml

```diff
@@ -0,0 +1,35 @@
+app-id: ca.potatoe.cmdjewel
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.llvm20
+command: cmdjewel
+finish-args:
+  - --socket=pulseaudio
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+  build-args:
+    - --share=network
```

Comment:
https://docs.flathub.org/docs/for-app-authors/requirements#no-network-access-during-build